flink-core/src/main/java/org/apache/flink/configuration/JobManagerOptions.java

```diff
@@ -488,6 +488,23 @@ public enum SchedulerType {
                                             code(SchedulerExecutionMode.REACTIVE.name()))
                                     .build());
 
+    @Documentation.Section(Documentation.Sections.EXPERT_SCHEDULING)
+    public static final ConfigOption<Duration> SCHEDULER_SCALING_INTERVAL_MIN =
+            key("jobmanager.adaptive-scheduler.scaling-interval.min")
+                    .durationType()
+                    .defaultValue(Duration.ofSeconds(30))
+                    // rescaling and let the user increase the value for high workloads
+                    .withDescription(
+                            "Determines the minimum time (in seconds) between scaling operations in reactive mode.");
```

Comment:
1. `in seconds` can be removed due to the type is duration instead of int. It can be various timeUnit.
2. Is the `in reactive mode` exact? As I understand, this option is related to adaptive scheduler event is the reactive mode is disable.

Reply:
1. if the parameter value is passed by CLI as a String, will the duration be parsed correctly with the correct timeUnit?
2. yes but new resources addition call that triggers the rescale (and the cooldown period) only happens in reactive mode.

Reply:
done

Reply:
>> Is the in reactive mode exact? As I understand, this option is related to adaptive scheduler even if the reactive mode is disable.
> 
>  2. yes but new resources addition call that triggers the rescale (and the cooldown period) only happens in reactive mode.

The rescale can happen when adaptive scheduler is enabled with reactive mode[1]. And starting from `flink-1.18`, the rescale can happen when adaptive scheduler is enabled with the rest api of FLIP-291 [1], right? 

[1] https://nightlies.apache.org/flink/flink-docs-release-1.18/docs/deployment/elastic_scaling/#reactive-mode
[2] https://cwiki.apache.org/confluence/display/FLINK/FLIP-291%3A+Externalized+Declarative+Resource+Management

Reply:
doc states "You can also use Adaptive Scheduler without Reactive Mode," fair enough adaptive scheduler and reactive mode can be decoupled and Executing#onNewResourceRequirements belongs to Adaptive Scheduler, but min and max parameters refer to rescaling which happens only when reactive mode is activated, no ?

Reply:
I guess the min and max parameters take effect when adaptive scheduler is enabled with the rest api of FLIP-291.

Because I don't see any logic that they're just enabled when reactive mode is activated. And these code is placed at adaptive scheduler, so I guess they take effect once adaptive scheduler is enabled. 

Or you can try it locally, enable adaptive scheduler without reactive mode, and rescale it with the rescale button of FLIP-291 in Flink WebUI.

Reply:
No progress? I see the description is still `in reactive mode`.

Reply:
fair enough, done